package.json

```diff
@@ -89,7 +89,11 @@
     "extends": "airbnb",
     "rules": {
       "react/jsx-filename-extension": "off",
-      "react/require-default-props": "off"
+      "react/require-default-props": "off",
+      "import/no-extraneous-dependencies": "off"
+    },
+    "env": {
+        "browser": true
```

Comment:
need to remove two extra spaces here,
[ident coding style for this project is 2 spaces](https://github.com/kriasoft/react-static-boilerplate/blob/5711655be48a1fffd1c9e787e440709ee73c57b8/.editorconfig#L9-L11)